interactions/api/models/channel.py

```diff
@@ -183,13 +183,13 @@ def __init__(self, **kwargs):
 
     async def send(
         self,
-        content: Optional[str] = None,
+        content: Optional[str] = MISSING,
         *,
-        tts: Optional[bool] = False,
+        tts: Optional[bool] = MISSING,
         # attachments: Optional[List[Any]] = None,  # TODO: post-v4: Replace with own file type.
-        embeds=None,
-        allowed_mentions=None,
-        components=None,
+        embeds=MISSING,
```

Comment:
These are missing the `Optional` typing annotation. Please add these to the argument signature